### PR TITLE
Add SPV_EXT_shader_atomic_float_min_max

### PR DESCRIPTION
**Pull request by jekstrand**

### CODE REVIEW

File: extensions/EXT/SPV_EXT_shader_atomic_float_min_max.asciidoc [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+----
+
+
+Modifications to the SPIR-V Specification, Version 1.5
+------------------------------------------------------
+
+Modify Section 3.31, "Capability", adding this row to the Capability table:
+
+--
+[cols="1^.^,15,15",options="header"]
```
Comment (AlexeySotkin) [inline extensions/EXT/SPV_EXT_shader_atomic_float_min_max.asciidoc:77]:
```suggestion
[cols="1,15,15",options="header"]
```

**Comment by johnkslang** [comment]
I think this is missing the entry into the index in the README.

**Comment by jekstrand** [comment]
> I think this is missing the entry into the index in the README.

Done.

**Comment by raunraun** [comment]
LGTM, merging

**Pull request merged**